src/main/java/org/openjdk/jextract/JextractTool.java

```diff
@@ -568,7 +621,10 @@ private Optional<Path> inferPlatformIncludePath() {
                 }
             }
         }
-
         return Optional.empty();
     }
+
+    private String formatFrameworkPath(String optionString) {
+        return String.format(":/System/Library/Frameworks/%1$s.framework/%1$s", optionString);
```

Comment:
Does this work if a custom framework dir is used? Wouldn't this absolute path point to the wrong place?